### PR TITLE
Update mispelling

### PR DESCRIPTION
**Pull request by tibabit**

### CODE REVIEW

File: source/keyboardHandler.py [hunk 1]
```diff
@@ -573,7 +573,7 @@ def getDisplayTextForIdentifier(cls, identifier):
 inputCore.registerGestureSource("kb", KeyboardInputGesture)
 
 def injectRawKeyboardInput(isPress, code, isExtended):
-	"""Injet raw input from a system keyboard that is not handled natively by Windows.
+	"""Inject raw input from a system keyboard that is not handled natively by Windows.
```
Comment (josephsl) [inline source/keyboardHandler.py]: @jcsteh, I think you know more about this line. Thanks.

Comment (josephsl) [inline source/keyboardHandler.py]: @jcsteh, I think you know more about this line. Thanks.

Comment (josephsl) [inline source/keyboardHandler.py]: @jcsteh, I think you know more about this line. Thanks.

**Review by jcsteh — APPROVED** [review]

**Comment by jcsteh** [comment]
Thanks. Merged directly to master, as it just fixes a typo in a docstring.

**Pull request merged**